Sources/Commands/PackageCommands/AddTarget.swift

```diff
@@ -185,6 +203,218 @@ extension SwiftPackageCommand {
                 }
             }
         }
+
+        private func createAuxiliaryFile(
+            fileSystem: any FileSystem,
+            rootPath: AbsolutePath,
+            filePath: RelativePath,
+            contents: SourceFileSyntax,
+            verbose: Bool = false
+        ) throws {
+            // If the file already exists, skip it.
+            let filePath = rootPath.appending(filePath)
+            if fileSystem.exists(filePath) {
+                if verbose {
+                    print("Skipping \(filePath.relative(to: rootPath)) because it already exists.")
+                }
+
+                return
+            }
+
+            // If the directory does not exist yet, create it.
+            let fileDir = filePath.parentDirectory
+            if !fileSystem.exists(fileDir) {
+                if verbose {
+                    print("Creating directory \(fileDir.relative(to: rootPath))...", terminator: "")
+                }
+
+                try fileSystem.createDirectory(fileDir, recursive: true)
+
+                if verbose {
+                    print(" done.")
+                }
+            }
+
+            // Write the file.
+            if verbose {
+                print("Writing \(filePath.relative(to: rootPath))...", terminator: "")
+            }
+
+            try fileSystem.writeFileContents(
+                filePath,
+                string: contents.description
+            )
+
+            if verbose {
+                print(" done.")
+            }
+        }
+
+        private func addAuxiliaryFiles(
```

Comment:
I'm assuming this is basically bringing back what was here before moving it out? I do wonder if this could be merged somehow with the `package init` at all. But if it's just bringing things back, then maybe it's fine to make this a later problem.

Reply:
Yeah, this is just moving things from one place to the other.